src/Core/src/Primitives/EvaluateJavaScriptAsyncRequest.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+
+namespace Microsoft.Maui
+{
+	public record EvaluateJavaScriptAsyncRequest(string Script, Action<string> SetResult);
```

Comment:
We need a way to propagate an exception, so maybe 2 Actions? or a combined `Action<string, Exception>`?

Reply:
If we know that this is always an async, then we can even pass the `TaskCompletionSource` in here directly. The the callback just sets the right result/error.